products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,100 @@
-    + Do you need to poll any APIs for status?
-    + Are you handling all failure and error cases while in custody of your users's data?
+  - Does the project introduce any new or unusual infrastructure dependencies?
+    - No
+  - Do you need to poll any APIs for status?
+    - No
+  - Are you handling all failure and error cases while in custody of your users's data?
+    - Yes
 - Internal API changes
-    + List new or modified APIs in `vets-api`
-    + Are you deprecating or removing any APIs?
-    + Do you have API documentation?
-    + Describe expected call patterns
+  - List new or modified APIs in `vets-api`
+    - `/v0/caregivers_assistance_claims/facilities` uses the Lighthouse facilities api endpoint
+  - Are you deprecating or removing any APIs?
+    - No
+  - Do you have API documentation?
+    - No, the only change is a new endpoint in the caregivers controller.
+  - Describe expected call patterns
+    - `vets-website` calls this endpoint to a list of paginated facilities
 - External API changes
-    + List new or modified APIs for upstream or external systems
-    + Describe expected call patterns
-    + What PII or PHI will be transmitted to/from the external systems?
+  - List new or modified APIs for upstream or external systems
+    - No Changes
+  - Describe expected call patterns
+    - No Changes
+  - What PII or PHI will be transmitted to/from the external systems?
+    - No Changes
+  - List any required background processing
+    - No Changes
+  - Describe error and dead letter handling
+    - No Changes
 - Background jobs
-    + List any required background processing
-    + Describe error and dead letter handling
+  - List any required background processing
+  - Describe error and dead letter handling
 - Data storage
-    + Describe new or modified databases, tables or columns
-    + Describe indexes and constraints
-    + Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+  - Describe new or modified databases, tables or columns
+    - No Changes. The facility that is ultimately returned is used to pass the facility id as part of the 1010CG Form submission.
+  - Describe indexes and constraints
+    - No Changes
+  - Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+    - No Changes
 - Libraries and dependencies
-    + List new or updated dependencies
+  - List new or updated dependencies
+    - Lighthouse Facilities API (New to the 10-10CG, but used in VA.gov)
+    - Mapbox API (New to the 10-10CG, but used in VA.gov)
 - Metrics, logging, observability, alerting
-    + Identify key areas to monitor
+  - Identify key areas to monitor
+    - Monitor api status for mapbox api and Lighthouse facilities api
 - Infrastructure and network changes
-    + List any changes or additions
+  - List any changes or additions
 - Test strategy
-    + Describe automated, manual and user acceptance test strategy
```

Comment:
@allanto-ah When you have some time would you be able to take a look at what some of our test strategy may be for this?

Reply:
The testing strategy would be:

- Automated
  - Unit tests and Cypress E2E tests are implemented
- Manual
  - We created test cases based on the requirements of the feature and Figma designs
  - We have a regression test suite that will ensure nothing else broke from the new change
  - All test cases are documented in TestRail
- User acceptance
  - After implementation and testing, we will review with the Product Manager to ensure that all requirements have been met

Reply:
I added these in.